internal/dsync/drwmutex.go

```diff
@@ -56,6 +56,14 @@ const drwMutexRefreshInterval = 10 * time.Second
 
 const drwMutexInfinite = 1<<63 - 1
 
+func sleepDuration(r *rand.Rand) time.Duration {
+	duration := time.Duration(r.Float64() * float64(lockRetryInterval))
+	if duration > lockRetryInterval {
```

Comment:
This is impossible:

> Float64 returns, as a float64, a pseudo-random number in the half-open interval [0.0,1.0).

AFAICT this PR does nothing.

Reply:
You are right, I need to debug what is happening here elsewhere.